Modules/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -1657,10 +1661,27 @@ private function addCensorshipObject()
         $this->handleCensorship();
     }
 
+    /**
+     * @return string
+     */
+    private function getModalActions()
+    {
+        $modalString = '';
+        foreach ((array) $this->modalActionsContainer as $modal) {
+            global $DIC;
```

Comment:
Please remove this `global` and initialize two properties for the renderer and the factory in `__construct`.